test/test-ncu.js

```diff
@@ -95,14 +94,21 @@ describe('npm-check-updates', function () {
         });
 
         it('should fall back to package.json search when receiving empty content on stdin', function (done) {
-            var childProcess = child.exec('node bin/ncu', function (error, stdout) {
-                if (error) {
-                    done(error);
-                }
+            spawn('node', ['bin/ncu']).then(function (stdout) {
```

Comment:
You would want to `return spawn(...)` and remove `done` from the function block so that the mocha can handle the test resolution automatically.